MATLAB/Algorithms/FDK.m

```diff
@@ -116,4 +121,23 @@
     end
 end
 
+end
+
+function proj = apply_wang_weights(proj, geo)
+
+    overlap_in_mm = geo.sDetector(1)/2 - abs(geo.offDetector(1));
+    overlap_in_pix = round(overlap_in_mm / geo.dDetector(1));
+
+    t_in_mm = linspace(-overlap_in_mm, overlap_in_mm, 2 * overlap_in_pix);
+    R_in_mm = geo.DSO;

```

Comment:
While geometries where DSO changes per-projection should not be used with FDK, they are still supported and they may give us a bug in the future if someone tries to run this with that. I suggest changing this to 

```
R_in_mm = geo.DSO(1);
```

Not necessarily correct for changing DSOs, but neither is using FDK and Wang weights, so whatever. It solves our possible bug. maybe we can leave a comment mentioning that.

Reply:
I completely agree, it worked for my situation but I'll fix this so it doesn't mess up any's code down the line.

Reply:
Great! Feel free to fix all these, otherwise I might find some time next week to apply them myself, if you can't do it. ;)